src/components/ScholarshipForm.jsx

```diff
@@ -114,6 +145,90 @@ function ScholarshipForm({ scholarship, submitFn, onSubmitError }) {
     </Grid>
   );
 
+  stepperItems['Eligibility Requirements'] = (
```

Comment:
Please guard this with an experiment. Or at least the grades and ethnicities ones. We haven't quite decided the data model for how to store these.

Reply:
Actually, you may want to guard the whole thing (or most of the fields) for now as some of the options are limited (e.g. the state dropdown doesn't have all 50 states).

Or you could add to `constants.ts` some of those as lists:
```ts
// Source: https://gist.github.com/bubblerun/a624de5b4fa8ff0980010054a7220977
const STATES = [...];

// Source: https://www.nomajordrama.co.nz/Majors
const MAJORS = [...];

// Source: https://www.university-list.net/us/universities-2000.htm
const COLLEGES = [...];
```
Feel free to use different sources and scrape things however you like (e.g. you could include the state for the college, the major's general area so that they're groupable and more easily selectable by general area, etc.)

Alternatively you could break up this PR into multiple ones and add a single field or a group of similar fields for each if you'd like.